src/main/groovy/io/saagie/plugin/dataops/clients/SaagieClient.groovy

```diff
@@ -1859,19 +1627,12 @@ class SaagieClient {
                 }
                 return listJobs
             }
-        } catch (InvalidUserDataException invalidUserDataException) {
-            throw invalidUserDataException
-        } catch (GradleException stopActionException) {
-            throw stopActionException
-        } catch (Exception exception) {
-            logger.error('Unknown error in projectsImport')
-            throw exception
-        }
+        }, 'Unknown error in projectsImport');
```

Comment:
'Unknown error in task projectsImport, function getJobListByNameAndId'

Reply:
Another mistake of name log